moulinette/authenticators/ldap.py

```diff
@@ -233,14 +233,33 @@ def validate_uniqueness(self, value_dict):
         Returns:
             Boolean | MoulinetteError
 
+        """
+        attr_found = self.get_conflict(value_dict)
+        if attr_found:
+            logger.info("attribute '%s' with value '%s' is not unique",
+                        attr_found[0], attr_found[1])
+            raise MoulinetteError(errno.EEXIST,
+                                  m18n.g('ldap_attribute_already_exists',
+                                         attribute=attr_found[0],
+                                         value=attr_found[1]))
```

Comment:
Uh in fact I don't understand how that changes the behavior compared to the current code o.O ? Am i missing something o.O ?

Reply:
I just adapted the function "validate_uniqueness" to don't duplique the code. I kept this function to keep the compatibility with the actual code in the core

Reply:
Okay so in fact now i understand the motivation of this PR which is (if i understood correctly) to factorize the get_conflict part so that you can use it in YunoHost core like in YunoHost/yunohost#585

Reply:
Yes :+1: